Sources/Extensions/Intents/Resources/Base.lproj/AppIntentVocabulary.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IntentPhrases</key>
+	<array>
+		<dict>
+			<key>IntentName</key>
+			<string>INShareFocusStatusIntent</string>
+			<key>IntentExamples</key>
+			<array>
+				<string>Share focus in home assistant</string>
```

Comment:
let's do `Home Assistant` even if we're not translating

Reply:
Done